lib/mayaUsd/ufe/UsdTransform3dRead.h

```diff
@@ -31,7 +32,9 @@ namespace ufe {
 // Note that all calls to specify time use the default time, but this
 // could be changed to use the current time, using getTime(path()).
 
-class MAYAUSD_CORE_PUBLIC UsdTransform3dRead : public Ufe::Transform3dRead
+class MAYAUSD_CORE_PUBLIC UsdTransform3dRead
+    : public UsdTransform3dReadImpl
```

Comment:
Shouldn't we be inheriting privately from UsdTransform3dReadImpl?  It's just a code sharing technique, there is no class hierarchy involved here.  Also, I worry about ambiguity if there are identically-named virtual methods in Ufe::Transform3dRead and UsdTransform3dReadImpl.